File: official/docs/php/current/referral-customers/add-payment-method-with-bank-account.php
<?php

$client = new \EasyPost\EasyPostClient(getenv('EASYPOST_API_KEY'));

$paymentMethod = $client->betaReferralCustomer->addPaymentMethod([
    'cus_...',
    'ba_...',
    'primary'
]);

echo $paymentMethod;

File: official/docs/ruby/current/referral-customers/add-payment-method-with-bank-account.rb
require 'easypost'

client = EasyPost::Client.new(api_key: ENV['EASYPOST_API_KEY'])

payment_method = client.beta_referral_customer.add_payment_method(
  'cus_...',
  'ba_...',
  'primary',
)

puts payment_method
